docs/core/tutorials/using-on-macos.md

```diff
@@ -49,95 +44,78 @@ terminal window, if you prefer).
 
 By the time we're done, you'll create three projects: a library project,
 tests for that library project, and a console application that makes
-use of the library. You'll follow a standard folder structure for
-the three projects. Following this standard folder structure
-means that the .NET Core SDK tools understand the relationship between
-your production code projects and your test code projects. That makes
-your development experience more productive.
+use of the library. 
 
 Let's start by creating those folders. In the terminal, create a 'golden'
```

Comment:
@spboyer Good catch. I made those changes. I also updated the closing to match the new VS Code extension behavior.